templates/possible.yaml

```diff
@@ -6,9 +6,9 @@ prompt: |-
 
   [BEGIN DATA]
   ************
-  [Task]: {{{input}}}
+  [Task]: {{input}}
```

Comment:
the triple backticks are important (they prevent HTML escaping)

Reply:
resolved offline